test/expect/TestScript.test_pretty_printer-if_one.expect

```diff
@@ -0,0 +1,7 @@
+  %2 = aten::lt(a, b)
```

Comment:
Not related to this patch, but the mixing of `%2` and `a` really makes my eyes bleed haha

Reply:
We could instead use some kind of generated temporary names (e.g. `t1`, `t2`, ...) for this or just remove the intermediate values altogether with something like `prim::TensorToNum(aten::lt(a, b))` but that could get long.